src/System.Diagnostics.Process/src/System/Diagnostics/ProcessManager.Linux.cs

```diff
@@ -41,25 +41,25 @@ private static ProcessInfo CreateProcessInfo(int pid)
                 Interop.procfs.ParsedStat procFsStat = Interop.procfs.ReadStatFile(pid);
                 pi = new ProcessInfo
                 {
-                    _processId = pid,
-                    _processName = procFsStat.comm,
-                    _basePriority = (int)procFsStat.nice,
-                    _virtualBytes = (long)procFsStat.vsize,
-                    _workingSet = procFsStat.rss,
-                    _sessionId = procFsStat.session,
-                    _handleCount = 0, // not a Unix concept
+                    ProcessId = pid,
+                    ProcessName = procFsStat.comm,
+                    BasePriority = (int)procFsStat.nice,
+                    VirtualBytes = (long)procFsStat.vsize,
+                    WorkingSet = procFsStat.rss,
+                    SessionId = procFsStat.session,
+                    HandleCount = 0, // not a Unix concept
 
                     // We don't currently fill in the following values.
                     // A few of these could probably be filled in from getrusage,
                     // but only for the current process or its children, not for
                     // arbitrary other processes.
-                    _poolPagedBytes = 0,
-                    _poolNonpagedBytes = 0,
-                    _virtualBytesPeak = 0,
-                    _workingSetPeak = 0,
-                    _pageFileBytes = 0,
-                    _pageFileBytesPeak = 0,
-                    _privateBytes = 0,
+                    PoolPagedBytes = 0,
```

Comment:
Now that we have a mechanism for throwing an exception if we don't have this data, I'd suggest we just remove all of these 0 values and instead let them be the -1 default so that attempting to access them will result in an exception.